### PR TITLE
Enable --static-swift-stdlib on Linux

### PR DESCRIPTION
**Pull request by keith**
https://bugs.swift.org/browse/SR-648 has been resolved, so we can enable
this flag on Linux

### CODE REVIEW

**Comment by dduan** [comment]
@swift-ci please smoke test

**Review by neonichu — APPROVED** [review]

**Comment by neonichu** [comment]
LGTM, @drexin would also be good if you could take a look.

**Review by drexin — APPROVED** [review]
LGTM, thanks!

**Comment by keith** [comment]
Thanks!

**Comment by keith** [comment]
If you think this should go in 5.3 let me know and I can submit a PR

**Comment by fabianfett** [comment]
@keith at least I would love to see this in 5.3.

**Comment by keith** [comment]
I submitted https://github.com/apple/swift-package-manager/pull/2922 in case the maintainers want this

**Pull request merged**
